csi/nodeserver.py

```diff
@@ -47,6 +49,27 @@ def NodePublishVolume(self, request, context):
             pvtype=pvtype
         ))
 
+        if not request.volume_id:
+            errmsg = "Volume ID is empty and must be provided"
+            logging.error(errmsg)
+            context.set_details(errmsg)
+            context.set_code(grpc.StatusCode.NOT_FOUND)
```

Comment:
If given volume_id doesn't exist it'll be `NOT_FOUND`, however it volume_id isn't supplied it'll be `INVALID_ARGUMENT`.

You can refer https://github.com/kubernetes-csi/csi-test/blob/v3.1.1/pkg/sanity/node.go#L360

Reply:
Thanks! I was referring https://github.com/container-storage-interface/spec before.